config.json

```diff
@@ -121,6 +121,7 @@
     "depth": 1,
     "message": "Spamming with HORNET tipselect",
     "tag": "HORNET99INTEGRATED99SPAMMER",
+    "maxCPUUsage": 0.5,
```

Comment:
Maybe `maxCpuUsage` looks better? I don't know :)

Reply:
Yes, the CPUU in the middle is hard to read. What I did is the rules/guidlines from "The Go Programming Language" book that say letter if acronyms and initialisms like ASCII and HTML are always rendered in the same case. So perhaps cpuMaxUsage ?

Reply:
cpuMaxUsage sounds good